docs/release-v0.9.md

```diff
@@ -0,0 +1,379 @@
+* updated CI and other scripts
+* updated Dockerfiles and compose definitions
+* updated cmd apps and their config
+* `go mod tidy` all modules (`task tidy`)
+* updated integration and acceptance test framework
+
+### Breaking Changes and Deprecations in CLI Applications
+
+* The `kwil-admin peer` command is deprecated, which like the `init` command, which initializes the configuration for a new node. The equivalent functionality of the `peer` command is now achieved with the `init` command by using the `--genesis` flag to provide a genesis.json file to use. In addition, the `init` command now accepts all of the node's flags to specify settings to use in the generated `config.toml`. ([d1cf754](https://github.com/kwilteam/kwil-db/commit/d1cf754c6be8c8a785487c36d7d21155a0a83033)
+
+* With `kwil-cli`, the `--action` flag used with the `database execute` and `database call` commands is now deprecated. Since this is a required input, it is now a positional argument (just omit `--action`). ([852df6a](https://github.com/kwilteam/kwil-db/commit/852df6a34cb14c9cb43e6bfb2f8e087cf6154798))
+
+* With `kwild`, the `rpc_req_limit` setting has been renamed to `rpc_max_req_size` to reflect that it is a size limit rather than a rate limit. `rpc_req_limit` is now deprecated. ([56a765d](https://github.com/kwilteam/kwil-db/commit/56a765d414aed37d183cbe8dc70f9ef7e74793d2))
+
+* The `reset` and `reset-state` subcommands of `kwil-admin setup` are reworked to also reset the PostgreSQL database. The `reset-state` command only resets PostgreSQL, while `reset` also deletes the data folders in the application's root directory. ([2f6b18e](https://github.com/kwilteam/kwil-db/commit/2f6b18ed1c08ddfc255542631ed9f23bbee3b6b0), [e876a07](https://github.com/kwilteam/kwil-db/commit/e876a07460e453642a1353d76e096b5a0cbf0175))
+
+* The `kwil-admin migrate genesis-state` command's `--root-dir` flag is renamed to `--out-dir`, and the default is now `~/.genesis-state` so as not to accidentally overwrite an existing genesis file if node is running locally. ([5dc5f27](https://github.com/kwilteam/kwil-db/commit/5dc5f2727928c46c289beb04d731196bd5e807f9))
+
+### Breaking Changes to Kuneiform and SQL Rules
+
+Please note the changes to Kuneiform and the parser in [Kuneiform Changes](#kuneiform-changes). Existing schemas that are migrated from an older version of kwild may have functional changes.
+
+### Breaking Changes to Transaction Execution
+
+* If a validator remove transaction targets a public key that is not in the validator set, the transaction fails (non-zero result code) rather than silently doing nothing.  Integration tests were already designed with this behavior in mind, but did not query status of the removal transaction on a node where it was executed. This is a breaking change to applications the expected the previous outcome. ([b8a52a3](https://github.com/kwilteam/kwil-db/commit/b8a52a3445e60052d3e6cb809cff3bf682aa9f9a))
+* ... anything else?
+
```

Comment:
I will create a PR that adds a breaking changes section for extensions, as there were several. They are easy to upgrade from, but need some explaining